examples/getting_started/3_Deep_dive_into_the_anatomy_of_quantum_circuits.ipynb

```diff
@@ -41,7 +41,6 @@
     "# magic word for producing visualizations in notebook\n",
```

Comment:
<p>Too complicated. I'd simplify to something like</p><p> </p><p>"This is possible because we only need to measure in at most one basis for each qubit. For instance in the example above, on qubit 2 we only measure in the Y basis."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>